### ISSUE TITLE
Using -f is not using a local file, throwing error.

### ISSUE THREAD

**Issue by dcp3450** [toddbluhm/env-cmd#359]
I'm have env-cmd installed locally on a windows 10 setup. Calling `env-cmd -f .env.local` returns the error:

`Error: Failed to find .env file at default paths: [./.env,./.env.js,./.env.json]`

**Comment by anichols-ht** [comment]
Ah, there's a separate option for this: `-r, --rc-file [path]`. With all due respect, I think that's a confusing implementation. It should just always use the `-f` flag. I can't see any reason you would need to pass _both_.

**Comment by toddbluhm** [comment]
@anichols-ht I agree with this and am considering just simplifying all the parameters. See this issue which is similar #372
